cloudigrade/config/settings/base.py

```diff
@@ -467,47 +490,57 @@ def __print(*args, **kwargs):
     "api.tasks.delete_from_sources_kafka_message": {
         "queue": "delete_from_sources_kafka_message"
     },
-    "api.tasks.update_from_sources_kafka_message": {
-        "queue": "update_from_sources_kafka_message"
+    "api.tasks.update_from_source_kafka_message": {
+        "queue": "update_from_source_kafka_message"
```

Comment:
Interesting. 🤔 How would you feel about instead renaming the underlying function to have plural `sources` instead of the singular `source`? That would make it more consistent with the other names, and that's probably what we intended before this bug.

Reply:
I don't mind updating in this PR, consistency is always good. 👍